bluepyopt/ephys/simulators.py

```diff
@@ -112,12 +112,19 @@ def neuron(self):
 
         return neuron
 
+    def check_voltage_valid(self, icell, voltage_bound=200.):
```

Comment:
I find this dangerous though. Ideally NEURON should detect if it gets values that it can't handle. Do we know what the effect this has on optimizations? I wouldn't be surprised if a lot of initial solutions might be out of bounds here.

Reply:
Ok,I will open an issue in Neuron

Reply:
Any news?

Reply:
The engineer who was looking into it is just back from vacation. He will resume working on it.